goblin/driver/cluster.py

```diff
@@ -156,13 +156,16 @@ def config_from_json(self, filename):
         """
         with open(filename, 'r') as f:
             config = json.load(f)
-        config = self._get_message_serializer(config)
+        config = self._process_config_imports(config)
         self.config.update(config)
 
-    def _get_message_serializer(self, config):
+    def _process_config_imports(self, config):
         message_serializer = config.get('message_serializer', '')
-        if message_serializer:
+        provider = config.get('provider')
+        if message_serializer and isinstance(message_serializer, str):
             config['message_serializer'] = my_import(message_serializer)
+        if provider and isinstance(provider, str):
+            config['provider'] = my_import(provider)
         return config
```

Comment:
Yeah that make sense. I suppose it will break if someone provides an empty string, but that will become obvious when they try to run the app.